std_msgs/String handedness
geometry_msgs/Point[] hand_landmarks
geometry_msgs/Point[] hand_world_landmarks
